x-pack/plugins/security_solution/public/landing_pages/components/landing_links_icons.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiFlexGrid,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiText,
+  EuiTitle,
+  IconType,
+} from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { SecurityPageName } from '../../app/types';
+import {
+  SecuritySolutionLinkAnchor,
+  withSecuritySolutionLink,
+} from '../../common/components/links';
+
+interface LandingLinksImagesProps {
+  items: NavItem[];
+}
+
+export interface NavItem {
+  id: SecurityPageName;
+  label: string;
+  icon: IconType;
+  description: string;
+  path?: string;
+}
+
+const Link = styled.a`
+  color: inherit;
+`;
+
+const SecuritySolutionLink = withSecuritySolutionLink(Link);
+
+const Description = styled(EuiFlexItem)`
+  max-width: 22em;
+`;
+
+const PrimatyEuiTitle = styled(EuiTitle)`
+  color: ${(props) => props.theme.eui.euiColorPrimary};
+
+  margin-top: ${({ theme }) => theme.eui.paddingSizes.m};
+  margin-bottom: ${({ theme }) => theme.eui.paddingSizes.xs};
+`;
+
+export const LandingLinksIcons: React.FC<LandingLinksImagesProps> = ({ items }) => (
+  <EuiFlexGrid columns={3} gutterSize="xl">
+    {items.map(({ label, description, path, id, icon }) => (
+      <EuiFlexItem key={id} data-test-subj="LandingItem">
+        <EuiFlexGroup
+          direction="column"
+          alignItems="flexStart"
+          gutterSize="none"
+          responsive={false}
+        >
+          <EuiFlexItem grow={false}>
+            <SecuritySolutionLink tabIndex={-1} deepLinkId={id} path={path}>
+              <EuiIcon aria-hidden="true" size="xl" type={icon} role="presentation" />
+            </SecuritySolutionLink>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <SecuritySolutionLinkAnchor deepLinkId={id} path={path}>
+              <PrimatyEuiTitle size="xxs">
```

Comment:
Wondering if it makes more sense to put the link inside the title instead, it should take the correct color. Also, there's a typo in `PrimatyEuiTitle`.

Reply:
Yes, it does make sense. Thanks!